<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>savepackagelistpage</class>
 <widget class="QWidget" name="savepackagelistpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>85</number>
     </property>
     <property name="bottomMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QTableView" name="savetableView">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>270</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="nextBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
